src/removeNPMAbsolutePaths.js

```diff
@@ -54,7 +54,12 @@ async function processFile(filePath, opts) {
 
     if (shouldWriteFile || opts.force) {
       try {
-        await writeFileAsync(filePath, JSON.stringify(obj, null, '  '));
+        const jsonStr = JSON.stringify(obj, null, '  ');
+        // insert newline if previously present
+        if (data.endsWith('\n')) {
+          jsonStr.concat('\n');
```

Comment:
Strings are immutable in javascript. `concat` doesn't change `jsonStr`, it just returns a new string. So this actually does nothing.